### ISSUE TITLE
Project Dead

### ISSUE THREAD

**Issue by ohdearaugustin** [justwatchcom/elasticsearch_exporter#349]
Is it still maintained? As I can see any contributions or comments from the core members @caarlos0 and @zwopir since the beginning of the year.

**Comment by zwopir** [comment]
Hey, not really dead, but in fact there hasn't been much work from my side. Too busy with other projects :(
I already asked a few times if there are people interested to contribute and help out but so far nobody volunteered.

The next steps I thought of are:
 - setup a build pipeline
 - switch from a stable, rc,  ... release model to a continuous model

It's really on my list to give this project some love. Sorry if this hasn't happened lately

**Comment by caarlos0** [comment]
@zwopir I can help with that :)

Been keeping my own fork for some time (eventually swiching back to the main one when changes get out), can also do the CI pipeline et al (I maintain goreleaser and have some exporters using it to release, can do that if you are into it)...

Anyway, the company I work for use this a lot, so I'm up to help maintaining it 👌

**Comment by ohdearaugustin** [comment]
@zwopir and @caarlos0 Thank you guys for your fast answers. I really appreciate it. If I can help you, just let me know.

@zwopir your next steps sound great.

@caarlos0 just gave your tool a short visit and I like your idea.

**Comment by roidelapluie** [comment]
Another approach is to move the repo to prometheus-community.

https://github.com/prometheus-community

We would then switch to the prometheus CI / build tools (promtool).

I could sponsor that move as I am a user of this exporter.

**Comment by caarlos0** [comment]
Sounds good too! Happy to help maintaining it anyway :)

**Comment by roidelapluie** [comment]
Created https://github.com/justwatchcom/elasticsearch_exporter/issues/349

**Comment by rtkgjacobs** [comment]
We use this component extensively as well internally. Have been considering adding CCS (Cross cluster search) metrics collections as a PR. Happy to contribute in part also perhaps, but community adoption seems most sensible.

**Comment by roidelapluie** [comment]
I have invited @caarlos0 and @zwopir to the prometheus-community org. Therefore the github repository can be transferred there.

**Comment by caarlos0** [comment]
sidenote for the sake of clarity: I just open some PRs/issues here, I'm not an official contributor (although I would be happy to be).

**Comment by roidelapluie** [comment]
Ack, I was mislead by the first comment.

If the repo is transferred, you will be on the team. It is @zwopir 's call to transfer or not the repository. If they have issue with it, they can comment here and we can further tune this.

**Comment by caarlos0** [comment]
> Ack, I was mislead by the first comment.

yeah, I figured, sorry about it...

> If the repo is transferred, you will be on the team. It is @zwopir 's call to transfer or not the repository. If they have issue with it, they can comment here and we can further tune this.

ack 👍

**Comment by ohdearaugustin** [comment]
@roidelapluie yeah sorry for the misleading first comment. I misinterpreted some commits to think @caarlos0 is also a core member. Thank you all of you guys for your efforts. (= Appreciate it.

**Comment by adrianmoisey** [comment]
I've just started using this exporter, and would love to see some movement on this project. 
@zwopir have you put any thought into moving it to the prometheus-community group?

Personally, I'd love to see more people getting involved, and see it moving forward a bit.

**Comment by infa-kparida** [comment]
This is one of the awesome exporters which still works and I am also using it.It would be nice if either it can be transferred to prometheus-community or the PR's can be reviewed for merging.Thanks @zwopir  for all the hard work.

**Comment by caarlos0** [comment]
fwiw, me and @sysadmind are working on this fork: https://github.com/caarlos0/elasticsearch_exporter

**Comment by simonfrey** [comment]
FYI: @caarlos0 We are currently in the process of moving this to prometheus-community :)

**Comment by roidelapluie** [comment]
While I can align with our CI etc we will need maintainers :)

**Comment by metalmatze** [comment]
The project was now moved to the prometheus-community org. All help welcome!

**Comment by ohdearaugustin** [comment]
Thank you for your work (=
